group_vars/all/vars.yml

```diff
@@ -102,4 +102,4 @@ django_db_host: "{{ application_db_host }}"
 
 # configure VMs to use local timezone
 # (requires including pul timezone role in playbook)
-timezone: America/New_York
+timezone: America/New_York
```

Comment:
editor removed the final newline? that's kinda weird

Reply:
I must have — I had a variable I added and then removed! I can fix.